doc/050_restore.rst

```diff
@@ -88,6 +88,22 @@ disk space. Note that the exact location of the holes can differ from those in
 the original file, as their location is determined while restoring and is not
 stored explicitly.
 
+Restoring extended file attributes
+----------------------------------
+
+By default, user namespaced extended attributes for files are restored on Linux,
+and all extended attributes are restored for other operating systems.
+
+Use ``--exclude-xattr`` and ``--include-xattr`` to control which extended
+attributes are restored for files in the snapshot. For example, to restore
+user and security namespaced extended attributes for files:
```

Comment:
Add `only` at the beginning of this line to make it clearer that specifying an `--include` limits the restored xattrs to the set matched by the filter. This is relevant with a default of restoring all xattrs.